File: docs/.vitepress/demos/gallery/items.ts
import type { PlGalleryItem } from 'plass-ui';

/** Six plates at four proportions, so a masonry has something to arrange. */
export const photos: PlGalleryItem[] = [
  {
    src: '/gallery-1.svg',
    alt: 'A harbour at dusk',
    title: 'Harbour',
    description: 'Busan',
    ratio: 4 / 3
  },
  {
    src: '/gallery-2.svg',
    alt: 'A bridge over a river',
    title: 'Bridge',
    description: 'Porto',
    ratio: 3 / 2
  },
  {
    src: '/gallery-3.svg',
    alt: 'A terraced hillside',
    title: 'Hillside',
    description: 'Banaue',
    ratio: 3 / 4
  },
  {
    src: '/gallery-4.svg',
    alt: 'A covered market',
    title: 'Market',
    description: 'Marrakesh',
    ratio: 1
  },
  {
    src: '/gallery-5.svg',
    alt: 'Dunes at first light',
    title: 'Dunes',
    description: 'Huacachina',
    ratio: 2
  },
  {
    src: '/gallery-6.svg',
    alt: 'A stepped terrace',
    title: 'Terrace',
    description: 'Lisbon',
    ratio: 2 / 3
  }
];

File: packages/flutter/example/lib/demos/gallery/items.dart
import 'package:flutter/widgets.dart';
import 'package:plass_ui/plass_ui.dart';

/// Six plates at four proportions, so a masonry has something to arrange.
final List<PlGalleryItem> photos = <PlGalleryItem>[
  PlGalleryItem(
    id: 'harbour',
    image: const NetworkImage('/gallery-1.svg'),
    semanticLabel: 'A harbour at dusk',
    title: 'Harbour',
    description: 'Busan',
    ratio: 4 / 3,
  ),
  PlGalleryItem(
    id: 'bridge',
    image: const NetworkImage('/gallery-2.svg'),
    semanticLabel: 'A bridge over a river',
    title: 'Bridge',
    description: 'Porto',
    ratio: 3 / 2,
  ),
  PlGalleryItem(
    id: 'hillside',
    image: const NetworkImage('/gallery-3.svg'),
    semanticLabel: 'A terraced hillside',
    title: 'Hillside',
    description: 'Banaue',
    ratio: 3 / 4,
  ),
  PlGalleryItem(
    id: 'market',
    image: const NetworkImage('/gallery-4.svg'),
    semanticLabel: 'A covered market',
    title: 'Market',
    description: 'Marrakesh',
    ratio: 1,
  ),
  PlGalleryItem(
    id: 'dunes',
    image: const NetworkImage('/gallery-5.svg'),
    semanticLabel: 'Dunes at first light',
    title: 'Dunes',
    description: 'Huacachina',
    ratio: 2,
  ),
  PlGalleryItem(
    id: 'terrace',
    image: const NetworkImage('/gallery-6.svg'),
    semanticLabel: 'A stepped terrace',
    title: 'Terrace',
    description: 'Lisbon',
    ratio: 2 / 3,
  ),
];
